module-types/FS/XL710BM1-4SP.yaml

```diff
@@ -0,0 +1,19 @@
+---
+manufacturer: Intel
```

Comment:
As this is an Intel card this should be located under `module-types/Intel`

Reply:
Fixed, thanks